maintainers/README.md

```diff
@@ -44,6 +44,10 @@ The team’s reason to exist is to make that principle discoverable and reproduc
   [@olafklingt]: https://github.com/olafklingt
   [@olaf]: https://discourse.nixos.org/u/olaf
 
+- Zach Mitchell (GitHub: [@zmitchell])
+
+  [@zmitchell]: https://github.com/zmitchell/
+
```

Comment:
```suggestion

- Brian McGee (Github: [@brianmcgee], Discourse: [@bmg])

  [@brianmcgee]: https://github.com/brianmcgee
  [@bmg]: https://discourse.nixos.org/u/bmg
```

Reply:
Hi @zmitchell, I meant to add myself to both. Shall I make a suggestion further down?

Reply:
There's a separate file for just the working group here: https://github.com/NixOS/nix.dev/pull/468/files#diff-aa46b0f178e55378224b0dd846934c5736fa2e4de6d18c20d2ab2d739f0f0567

You can make a suggestion to add yourself and I'll just accept it